chain/vm/runtime.go

```diff
@@ -51,8 +52,7 @@ func (m *Message) ValueReceived() abi.TokenAmount {
 	return m.msg.Value
 }
 
-// EnableGasTracing, if true, outputs gas tracing in execution traces.
-var EnableGasTracing = false
+const EnableGasTracingEnvVar = "LOTUS_VM_ENABLE_GAS_TRACING_VERY_SLOW"
```

Comment:
I'd do
```suggestion
// EnableGasTracing, if true, outputs gas tracing in execution traces.
var EnableGasTracing = os.Getenv("LOTUS_VM_ENABLE_GAS_TRACING_VERY_SLOW") == "1"
```

So that we don't have to lookup the env var on every gas charge

Reply:
Changed, thanks that makes it a lot simpler.